containers/proxy-squid-image/Dockerfile

```diff
@@ -30,6 +30,9 @@ COPY squid.conf /etc/squid/squid.conf
 RUN chown squid:squid /etc/squid/squid.conf
 RUN mkdir -p /run/squid && chown squid:squid /run/squid
 
+RUN mkdir -p /etc/squid/conf.d
+COPY squid_image.conf /etc/squid/conf.d/squid_image.conf
```

Comment:
the folder needs to have at least one file. This is why you are including this empty file?